cypress/support/step_definitions/common/i_see_string_int_times.js

```diff
@@ -0,0 +1,3 @@
+Then("I see {string} {int} times", (string, int) => {
+  cy.findAllByText(string, { exact: false }).should("have.length", int);
```

Comment:
Would `getAllByRole` not have worked here?

Reply:
I replicated another test (I see {string}) which is quite a loose test to just speed up writing tests, but yeah role would've worked. That was me being lazy :)